sdk/ml/azure-ai-ml/tests/component/e2etests/test_component.py

```diff
@@ -441,6 +442,7 @@ def test_component_create_default_code(self, client: MLClient, randstr: Callable
         assert component_resource2.display_name == display_name
 
     @pytest.mark.disable_mock_code_hash
+    @pytest.mark.skipif(condition=not is_live(), reason="reuse test, target to verify service-side behavior")
```

Comment:
why do we say this is a reuse test?

Reply:
I just forget to rename this, as other tests are all reuse...will rename this

Reply:
remove this `skipif` to make this test run both in placyback mode and live mode.